lib/Plack/Middleware/AccessLog/Structured.pm

```diff
@@ -36,13 +34,13 @@ sub new {
 sub call {
 	my ($self, $env) = @_;
 
-	my $t_before = Time::HiRes::time();
+	my $t_before = Time::Moment->now;
```

Comment:
I think that should be `->now_utc`, otherwise, this uses the system time zone (`DateTime` uses UTC by default in `->now()`, so `Time::Moment` has a different API here) and thus the `date` in the log message might be written with the wrong time zone (`Z` stands for UTC, if I remember correctly).

Reply:
Done in update